dynamic-layers/openembedded-layer/recipes-devtools/python/python3-adafruit-blinka_6.2.2.bb

```diff
@@ -11,14 +11,8 @@ S = "${WORKDIR}/git"
 inherit setuptools3
 
 DEPENDS += "python3-setuptools-scm-native"
-
-do_install:append() {
-# it ships ./bcm283x/pulseio/libgpiod_pulsein which is a prebuilt
-# 32bit binary therefore we should make this specific to 32bit rpi machines (based on bcm283x) only
-    if [ ${@bb.utils.contains('TUNE_FEATURES', 'callconvention-hard', '1', '0', d)} = "0" ]; then
-        rm -rf ${D}${PYTHON_SITEPACKAGES_DIR}/adafruit_blinka/microcontroller/bcm283x
-    fi
-}
+# to disable qa errors when shipping 32bit bynaries in 64bit rpi machines
+do_package_qa[noexec] = "1"
```

Comment:
This solves the problem, but does that by throwing a blanket on a QA task. I would look for a better solution here.

Reply:
what will that get us ?